src/test/resources/schemas/valid-schemas/com.maxmind/anonymous_ip/jsonschema/1-0-0

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I just looked for a schema [in iglu-central](https://github.com/snowplow/iglu-central/blob/master/schemas/com.maxmind/anonymous_ip/jsonschema/1-0-0) that was pretty simple and linted successfully when I was trying out my changes but very happy to choose a different test case if appropriate